PRISM_maproom/layout.py

```diff
@@ -283,6 +290,18 @@ def controls_layout():
                 """
                 )
             ),
+            html.H5("Dataset Sources in the IRI DL"),
+            html.P(
+                dcc.Markdown(
```

Comment:
`dcc.Markdown` looks handy. Instead of several `Markdown` interspersed with `H5` and `P`, we could put all this text in a single `Markdown`. (Not before the demo.)